addons/account_edi_facturx/models/account_edi_format.py

```diff
@@ -30,7 +30,7 @@ def _post_invoice_edi(self, invoices, test_mode=False):
         res = {}
         for invoice in invoices:
             attachment = self._export_facturx(invoice)
-            res[invoice] = {'attachment': attachment}
+            res[invoice] = {'success': True, 'attachment': attachment}
```

Comment:
@bfr-o Are you sure you want to have to put success True here even if the format does not need webservices?

Reply:
Yes, success is not related to the web-service.

NB: the flow of 'post' is slightly modified in this PR (separate the successful posting from the return of an attachment, rather use a special key, like for the cancel flow, could be useful). I give more details in the commit message.